core-processor/src/common.rs

```diff
@@ -483,7 +483,9 @@ impl ActorExecutionErrorReplyReason {
                 TrapExplanation::GasLimitExceeded => SimpleExecutionError::RanOutOfGas,
                 TrapExplanation::ForbiddenFunction => SimpleExecutionError::BackendError,
                 TrapExplanation::ProgramAllocOutOfBounds => SimpleExecutionError::MemoryOverflow,
-                TrapExplanation::Ext(_err) => SimpleExecutionError::BackendError,
+                TrapExplanation::InfallibleExt(_) | TrapExplanation::FallibleExt(_) => {
```

Comment:
Seems that we never can ran into FallibleExt here? :/
So trap explanation shouldn't contain this variant

Reply:
We can run either by `map_err` in `funcs.rs` or by `From` impl